bundles/org.openhab.binding.daikin/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -147,6 +173,172 @@
 			</options>
 		</state>
 	</channel-type>
+	
+	<channel-type id="acunit-cmpfrequency" advanced="true">
+		<item-type>Number</item-type>
+		<label>Compressor Frequency</label>
+		<description>Current compressor frequency</description>
+		<state readOnly="true" pattern="%.0f%%"></state>
+	</channel-type>
+	<channel-type id="acunit-specialmode" advanced="true">
+		<item-type>String</item-type>
+		<label>Special mode</label>
```

Comment:
The first letter of every word in a label should be capitalized.

Reply:
Fixed.